source/Calamari.Common/Plumbing/Deployment/PackageRetention/CaseInsensitiveTinyType.cs

```diff
@@ -4,63 +4,39 @@
 
 namespace Calamari.Deployment.PackageRetention
 {
-    //TODO: Replace this basic tiny types implementation with the Octopus one.
-    public abstract class CaseInsensitiveTinyType
+    public abstract class CaseInsensitiveTinyType : TinyType<string>
```

Comment:
This should be a no-op once https://github.com/OctopusDeploy/Calamari/pull/801 is merged.